src/js/utils/window_watchers.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/*
+  This utility is intended to communicate between chrome windows.
+  One window could watch passed values from another window and react to them.
+*/
+
+var windowWatcherUtil = {};
+
+windowWatcherUtil.invokeWatcher = function(bindingKey, bindingVal, watchersObject) {
```

Comment:
@Anisotropic You should change tabs to 4 spaces in this file as well.

Reply:
Done. It was "beautifier" for Atom..